scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/util/TaskLoggerRelativePathGenerator.java

```diff
@@ -42,6 +43,24 @@ public TaskLoggerRelativePathGenerator(TaskId taskId) {
         this.relativePath = taskId.getJobId().toString() + "/" + fileName;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this hard-coded path-delimiter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)

Reply:
Can you replace this code `"/"` with `File.seperator`?

Reply:
no because, this relative path is used as a relative path, and also as a dataspace file transfer directive (see TaskLauncher).

Relative paths on windows supports the slash, example new File(root,"some/relative/path") works.
File transfer directives only support slash and not backslash.

Reply:
Ok, very good to know, thanks!